lib/arel_toolkit/railtie.rb

```diff
@@ -0,0 +1,9 @@
+module ArelToolkit
+  class Railtie < ::Rails::Railtie
+    initializer 'ArelToolkit.insert' do
+      ActiveSupport.on_load :active_record do
+        Arel::Middleware::Railtie.insert_postgresql
```

Comment:
@mvgijssel why does this middleware lives in `Arel` and not in `ArelToolkit`?

Reply:
Everything is an addition to the Arel namespace, the ArelToolkit namespace is there for naming purposes of the gem.